.github/actions/clp-core-build-containers/action.yaml

```diff
@@ -46,11 +46,16 @@ runs:
           echo "BRANCH=latest" >> "$GITHUB_OUTPUT"
         fi
 
+    - id: "generate_id"
+      shell: "bash"
+      run: |
+        echo "ID=$(echo '${{github.repository}}' | tr '[:upper:]' '[:lower:]')" >> "$GITHUB_OUTPUT"
```

Comment:
* Can we move this into the previous step (`get_deps_image_prosp`)?
* Can we call the env var `REPOSITORY`?